### DISCUSSION TITLE
seperate 'bloom' setting from 'glow'' in environment

### DISCUSSION THREAD

**Discussion by djmaesen** [godotengine/godot-proposals#4674] [3D]
i only want to use 'bloom' in my project and not 'glow' in the environment setup.
however i need to set glow values in order to get some bloom. in some cases this lights up my entire model while i only want emissive materials to 'bloom' 
could it be possible to make 'bloom' its own setting in the environment setup?

**Comment by Calinou** [comment]
> however i need to set glow values in order to get some bloom. in some cases this lights up my entire model while i only want emissive materials to 'bloom'

This is the default behavior of glow if Bloom is set to 0.0. You can adjust the HDR Threshold and HDR Luminance Cap properties if you need further control over this.

> could it be possible to make 'bloom' its own setting in the environment setup?

Bloom is already its own property in the Environment resource – this has been the case since Godot's earliest open source version :slightly_smiling_face:

**Reply by djmaesen** [reply]
ok, close my suggestion, ive managed to get my environment setup working properly
